### PR TITLE
Charon project status

### PR DESCRIPTION
**Pull request by johandahlberg**
This converts the script create by @b97pla to get a project status update from charon to a st2 action and adds a rule to run that action every Monday morning.

### CODE REVIEW

**Comment by johandahlberg** [comment]
@johanherman This is ready for review.

**Comment by johanherman** [comment]
Glorified crontab huh? :P

File: config.yaml [hunk 1]
```diff
@@ -34,3 +34,8 @@ siswrap_service_port: 10700
 runfolder_svc_urls:
   - http://testarteria1:10800
 
+charon_api_token: dummy_token
```
Comment (johanherman) [inline config.yaml]: Corresponding PR for SNPSEQ repo coming I presume.

Comment (johandahlberg) [inline config.yaml]: Yes!

**Pull request merged**